screenfetch-dev

```diff
@@ -1381,6 +1381,11 @@ detectdisk () {
 }
 # Disk Usage Detection - End
 
+# Root Usage Detection - Begin
+detectrootusage(){
+ root=$(df -hT -xdevtmpfs -xtmpfs|sed -n 2p|awk '{print $4 "/" $3 " " $2}')
```

Comment:
The arguments for df used here are specific to GNU df and causes errors on systems that use BSD df (OS X). Should either wrap the usage of `detectrootusage` inside a conditional that checks that the version of df being used is part of GNU coreutils or provide a check for this inside of the `detectrootusage` function and provide an alternative for systems using BSD df.